src/server/api/controllers/users.controller.js

```diff
@@ -8,14 +9,39 @@ const {
 
 const { isInteger } = require('../lib/utils/validators');
 
-const getUsers = () => {
-  return knex('users').select(
-    'users.id',
-    'users.fullName',
-    'users.email',
-    'users.createdOn',
-    'users.firebaseUId',
-  );
+const getUsers = async () => {
+  const result = await client.search({
+    index: usersIndex,
+    body: {
+      query: {
+        match_all: {},
+      },
+      size: 50,
+      sort: [
+        {
+          fullName: {
+            order: 'asc',
+          },
+        },
+      ],
+    },
+  });
+  /* eslint no-underscore-dangle: ["error", { "allow": ["_source", "_id"] }] */
+  return result.body.hits.hits.map((hit) => ({ ...hit._source, id: hit._id }));
```

Comment:
It looks you are doing the same transformation in 2 places; you could extract that to a function or combined the functions into 1.